packages/cubejs-druid-driver/src/DruidQuery.ts

```diff
@@ -52,4 +52,8 @@ export class DruidQuery extends BaseQuery {
   public nowTimestampSql(): string {
     return `CURRENT_TIMESTAMP`;
   }
+
+  public countDistinctApprox(sql) {
+    return `APPROX_COUNT_DISTINCT_DS_THETA(${sql})`;
```

Comment:
Interesting to see that Druid supports 3 different implementations of HyperLogLog

-`APPROX_COUNT_DISTINCT` - Counts distinct values of a regular column or a prebuilt sketch column.
-`APPROX_COUNT_DISTINCT_DS_HLL` - Counts distinct values of an HLL sketch column or a regular column.
-`APPROX_COUNT_DISTINCT_DS_THETA` - Counts distinct values of a Theta sketch column or a regular column.

Reply:
https://druid.apache.org/docs/latest/querying/sql-aggregations.html

> The specific algorithm depends on the value of [druid.sql.approxCountDistinct.function](https://druid.apache.org/docs/latest/configuration/index.html#sql). By default, this is APPROX_COUNT_DISTINCT_BUILTIN. If the [DataSketches extension](https://druid.apache.org/docs/latest/development/extensions-core/datasketches-extension.html) is loaded, you can set it to APPROX_COUNT_DISTINCT_DS_HLL or APPROX_COUNT_DISTINCT_DS_THETA.

It looks like we need to use `APPROX_COUNT_DISTINCT`.

@tquachbot Do you have any reason why you specified directly `APPROX_COUNT_DISTINCT_DS_THETA` instead of using config?

Reply:
I work with Tim, but we personally use DS_THETA for increased accuracy of theta intersect operations. If there's an easy way to choose the approx distinct method through config that'd be ideal.

Reply:
Specifically how would we use config to specify different types of count distinct? Is it possible to change the count distinct definitions through some schema flag instead of having a single hardcoded option?